paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityShouldBurn.java

```diff
@@ -14,16 +16,27 @@ public class EntityShouldBurn extends EntityProperty<ElementTag> {
     // @input ElementTag(Boolean)
     // @plugin Paper
     // @description
-    // If the entity is a Zombie, controls whether it should burn in daylight.
+    // If the entity is a Zombie, Skeleton, or Phantom, controls whether it should burn in daylight.
     // -->
 
     public static boolean describes(EntityTag entity) {
-        return entity.getBukkitEntity() instanceof Zombie;
+        return entity.getBukkitEntity() instanceof Zombie
+                || entity.getBukkitEntity() instanceof Skeleton
+                || entity.getBukkitEntity() instanceof Phantom;
     }
 
     @Override
     public ElementTag getPropertyValue() {
-        return new ElementTag(as(Zombie.class).shouldBurnInDay());
+        if (getEntity() instanceof Zombie zombie) {
+            return new ElementTag(zombie.shouldBurnInDay());
+        }
+        else if (getEntity() instanceof Skeleton skeleton) {
+            return new ElementTag(skeleton.shouldBurnInDay());
+        }
+        else if (getEntity() instanceof Phantom phantom) {
+            return new ElementTag(phantom.shouldBurnInDay());
+        }
+        return null;

```

Comment:
Since the entity is guaranteed to be one of these 3 types, can replace the last check and `return null` with just `return as(Phantom.class).should...` (same for setting the value), see for [example](https://github.com/DenizenScript/Denizen/blob/ae22abdb46de2a34ce47c5e1d26261abda24d4f6/plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityWidth.java#L26-L31).